content/blog/2019-01-06-babette.md

```diff
@@ -0,0 +1,205 @@
+stopifnot(file.exists(fasta_filename))
+```
+
+To obtain a BEAST2 posterior with `babette`, do:
+
+```
+output <- bbt_run(fasta_filename)
+```
+
+And plotting the posterior trees:
+
+```
+plot_densitree(
+  output$alignment_trees,
+  alpha = 0.01,
+  consensus = as.character(c(1:4)),
+  cex = 2.0,
+  scaleX = TRUE,
+  scale.bar = FALSE
+)
+```
+
+Note that for plotting the posterior trees, I've added many arguments to 
+make it a beautiful blog post picture :-)
+
+#### More about babette
+
+`babette` actually is a suite of five packages, 
+calling and/or offering an alternative for the BEAST2 tool suite:
+
+What|BEAST2 suite|babette suite
+---|---|---
+Create a BEAST2 input file|BEAUti|`beautier`
+Run (only) BEAST2|`beast`|`beastier`
+Visualize posterior parameter estimates|Tracer|`tracerer`
+Visualize posterior phylogenies|DensiTree|`babette`
+Install BEAST2 packages|BEAUti|`mauricer`
+
+The BEAST2 tool suite has - at the moment - more functionality than `babette`.
+The [`babette` GitHub page](https://github.com/ropensci/babette) has a place where feature requests can be and
+are made. If you want something added and you volunteer to help (that is,
+answer my naive questions and testing that feature), let me know!
+
+Although `babette` may not have all BEAST2 functionality,
+`babette` already helped me create thousands of posteriors from an R script!
+Also the analysis done in this blog post, can be found
+at the [`babette` GitHub repository](https://github.com/ropensci/babette/blob/master/scripts/ropensci_create_figs_blog_post.R).
+
+#### Why rOpenSci
+
+I enjoy having my academic manuscripts reviewed.
+We expect the articles we read to be clear and the opinion
+of our peers help achieve this for our own work.
+But when I read/run the code in the unreviewed 
+supplementary materials section, there is a loss in
+clarity. Same goes for R packages written by academics:
+the article that describes these packages are well-written,
+but the actual code can be cumbersome to work with.
+To set a good example, I wanted my code reviewed.
+
+I enjoy having my code reviewed. Even when trying to
+write exemplary and spotless code, reviewers usually 
+expose the weaker spots in the software.
+My rOpenSci reviewers did an excellent job at that indeed! 
+Thanks to them, I think `babette` went from a good package to
+an excellent one. 
+
+### Acknowledgements
+
+Yacine Ben Chehida and Paul van Els supplied the first use cases.
+Reviews were provided by [David Winter](https://ropensci.org/authors/david-winter/) 
+and [Joëlle Barido-Sottani](https://github.com/bjoelle), 
+with a contribution from [Guangchuang Yu](https://guangchuangyu.github.io/).
+Raphael Scherrer and Jana Riederer helped by submitting bug reports. 
+
+### References
+
+[^1]: Bilderbeek, Richel JC, and Rampal S. Etienne. "babette: BEAUti 2, BEAST 2 and Tracer for R." Methods in Ecology and Evolution (2018). [doi.org/10.1111/2041-210X.13032](https://doi.org/10.1111/2041-210X.13032)
```

Comment:
In preview (https://deploy-preview-415--ropensci.netlify.com/blog/2019/01/06/babette/#fn:1), looks like `1111/2041-` in DOI is interpreted as math. Can you fix by escaping the `/`?